### ISSUE TITLE
[Bug] The Things Stack Industries Integration is frozen

### ISSUE THREAD

**Issue by Butanding** [thingsboard/thingsboard#5680]
**Describe the bug**
I want to add a new Integration of Type "The Things Stack Industries". I select the type and instantly the window is frozen.
Type Community and all the other Types are working correctly. But this type is not.
Tried it on all Devices, but it is the same. Seems like a Server error.

**Your Server Environment**
* own setup
  * cloud or local infrastructure or docker deployment: One Click Deploy with MS Azure
  * ThingsBoard Version: 3.3

**Your Client Environment**
OS: Ubuntu 18x with Browser Chrome and Firefox
Android with Chrome Browser

**To Reproduce**
See Introduction

**Expected behavior**
I want to add integration. No frozen window.

**Screenshots**
![image](https://user-images.githubusercontent.com/23176308/144628735-bcda88d3-001c-48ba-be7f-8608c2417968.png)

**Additional context**
Browser Console Log:
ERROR TypeError: t.form.get(...) is null

**Comment by JacksonBowe** [comment]
Hey @Butanding , I'm pretty sure this was an issue specifically related to v3.3.1

I also experienced this (fixed for me as of v3.3.2).

If for some reason you don't want to upgrade to v3.3.2 then you can create the integration through the API (this is what I did). Rough steps below:
- Navigate to https://{YOUR_DOMAIN}/swagger-ui.html
- Authorize (top right). You'll need your access token (refer [here](https://thingsboard.io/docs/reference/rest-api/))
- In the Authorize box write `Bearer {TOKEN}` -> OK
- Open the Integration Controller -> saveIntegration dropdowns
- Look at the example input and model values, select TTI for the integration type.

That's all I can remember off the top of my head, should give you a starting point.

**Comment by JacksonBowe** [comment]
@Butanding I just tried to create a new Things Industries integration and concur with the issue you have mentioned.

The "downlink topic pattern" field is `readonly`. Below is the result of inspecting the element.
```
<input _ngcontent-dyx-c755="" matinput="" required="" class="mat-input-element mat-form-field-autofill-control ng-tns-c175-57 ng-pristine cdk-text-field-autofill-monitored ng-untouched" id="mat-input-14" readonly="true" aria-invalid="false" aria-required="true" disabled="">
```

Here's what you can try to fix it. When you have this screen:
![image](https://user-images.githubusercontent.com/26427343/145290579-013dfacf-fb6b-4ec8-9a6f-eea302a00634.png)

Hit Inspect, find the <input> component I linked above and delete the `readonly="true"` part.

**Comment by Butanding** [comment]
@JacksonBowe 

Thanks again for your hint! It works if you edit the website manually.

But, stupid as I am, I don't know what to enter as a "Topic". I found out through the documentary from TTS: "{topic}" needs to be entered.

I finally made it. But I guess, it is a bug - or am I wrong?

Thanks again for your help!

**Comment by ashaforost** [comment]
Hi @Butanding @JacksonBowe 

We've fixed this bug in v3.3.1.
Please see our [Release Notes](https://thingsboard.io/docs/pe/reference/releases/#v331-september-3-2021) and [Upgrade Instructions](https://thingsboard.io/docs/user-guide/install/pe/upgrade-instructions/) articles.
